src/pages/strategies/create/recurring.test.tsx

```diff
@@ -0,0 +1,33 @@
+import { test, expect, afterEach } from 'vitest';
+import { cleanup, renderWithRouter, screen } from 'libs/testing-library';
+import { debugTokens } from '../../../../e2e/utils/types';
+import { CreateRecurringStrategyPage } from './recurring';
+
+const basePath = '/strategies/create/recurring';
+afterEach(async () => {
+  cleanup(); // Clear the screen after each test
```

Comment:
no need to call cleanup

Reply:
Vitest doesn't automatically call clean-up. 

https://github.com/vitest-dev/vitest/issues/1430

Tried changing the isolate and thread options but still ran into issues where rendering was adding elements without cleaning up between tests